CHANGELOG.md

```diff
@@ -6,6 +6,14 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- `hide` prop to allow hiding a LineChart axis
+- `type` prop to allow different interpolations of the LineChart
+- `tick` prop to allow a customized label on LineChart
+
+### Changed
+- `axis` prop to `xAxis` and `yAxis` props on LineChart schema
+- default `container` prop to have a `width` and `length` dependent of the screen size
```

Comment:
`width` and `length` or the second one should be `height`?

Reply:
Opss, you're right :dizzy_face: